docs/cli/data_format.rst

```diff
@@ -75,7 +35,7 @@ Without ID column::
 .. _libsvm-format:
 
 LibSVM Format
-^^^^^^^^^^^^^
+-------------
 
 The LibSVM format is a format for labels and sparse numerical
 features. It combines them in a single file,
```

Comment:
make similar changes as the libmultilabel part

Reply:
why not saying that

The format of each line is

<label1>,<label2>...,<labelL> <index1>:<value1> <index2>:<value2> ...